pnpm-lock.yaml

```diff
@@ -15838,7 +15847,7 @@ snapshots:
   '@babel/plugin-bugfix-safari-id-destructuring-collision-in-function-expression@7.22.15(@babel/core@7.22.5)':
     dependencies:
       '@babel/core': 7.22.5
-      '@babel/helper-plugin-utils': 7.22.5
+      '@babel/helper-plugin-utils': 7.25.7
```

Comment:
I don't know why `pnpm add -D -w @babel/plugin-syntax-explicit-resource-management@7.25.7` would change this dependency version `@babel/plugin-syntax-explicit-resource-management` doesn't even use `@babel/helper-plugin-utils`